### PR TITLE
Integration tests part 2

### PR DESCRIPTION
**Pull request by harrysarson**
A few further improvements to the integration tests landed in #84 plus adding three more integration tests.

The third is failing and will continue to fail untill a fix for #76 lands. The fix for ~76 will then need to include this patch:

```patch
diff --git a/integration-tests/typecheck/let-body-visibility/test.test.js b/integration-tests/typecheck/let-body-visibility/test.test.js
index 21f5d6e..3b239e6 100644
--- a/integration-tests/typecheck/let-body-visibility/test.test.js
+++ b/integration-tests/typecheck/let-body-visibility/test.test.js
@@ -1,7 +1,7 @@
 const test = require('ava');
 const {exec} = require('../../setup');
 
-test.failing(
+test(
     'typecheck: the let body can see the let bindings',
     exec,
     __dirname,
```

### CODE REVIEW

**Comment by harrysarson** [comment]
@Janiczek ready for review!

**Comment by Janiczek** [comment]
Hello @harrysarson sorry that this is taking so long. I'll try to get to it soon :crossed_fingers:

**Comment by harrysarson** [comment]
Hello @Janiczek. Not a problem! Take care and have a nice afternoon :)

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,11 @@
+
+# elm-format does not support exlucding files so we explicitly list all the directories
```
Comment (Janiczek) [inline Makefile:2]:
```suggestion
# elm-format does not support excluding files so we explicitly list all the directories
```

**Pull request merged**